tests/test_testPipeline.py

```diff
@@ -347,9 +380,13 @@ def testMockDiaPipelineTask(self):
         task = MockDiaPipelineTask(config=config)
         pipelineTests.assertValidInitOutput(task)
         result = task.run(pandas.DataFrame(), pandas.DataFrame(), afwImage.ExposureF(),
-                          afwImage.ExposureF(), afwImage.ExposureF(), 42, 'k')
+                          afwImage.ExposureF(), afwImage.ExposureF(), pandas.DataFrame(),
+                          pandas.DataFrame(), pandas.DataFrame(), 'k', 42)
```

Comment:
I think the `DiaPipelineTask` API has gotten to the point where we should be using keywords. 😬

Reply:
[Same question](https://github.com/lsst/ap_association/pull/230/files#r1706125095) about why this wasn't caught earlier. `MockDiaPipelineTask` allows `idGenerator=None`, but [the original does not](https://github.com/lsst/ap_association/blob/main/python/lsst/ap/association/diaPipe.py#L392).

(And again, such a fix should have been a separate commit.)

Reply:
I'll clean up some of the mocks, but it looks like they were not updated when we removed `ccdExposureIdBits`. This would seem to be an argument in favor of removing `test_testPipeline`, since it doesn't necessarily catch changes to Task API's unless the Mocks in `testPipeline.py` are also updated.